recipes/glib/all/conandata.yml

```diff
@@ -1,55 +1,58 @@
 sources:
-  "2.65.0":
```

Comment:
Don't need to comment versions in conandata.yml. The CI bot only reads config.yml to decide what will be built.

Reply:
I thought there was a hook checking precisely this, to avoid forgetting ?

Reply:
True! Hook 52 to be more exact.